dissect/target/plugins/apps/browser/chromium.py

```diff
@@ -625,12 +634,22 @@ def decrypt_v10(encrypted_password: bytes) -> str:
 
 
 def decrypt_v10_2(encrypted_password: bytes, key: bytes) -> str:
-    """
-    struct chrome_pass {
-        byte signature[3] = 'v10';
-        byte iv[12];
-        byte ciphertext[EOF];
-    }
+    """Decrypt a version 10 type 2 password.
+
```

Comment:
```suggestion

References:
```
Not sure if this behaves as expected.